commons/src/main/java/org/infinispan/commons/configuration/Json.java

```diff
@@ -0,0 +1,3247 @@
+   }
+
+   /**
+    * <p>
+    * Add the specified <code>Json</code> element to this array.
+    * </p>
+    *
+    * @return this
+    */
+   public Json add(Json el) {
+      throw new UnsupportedOperationException();
+   }
+
+   /**
+    * <p>
+    * Add an arbitrary Java object to this <code>Json</code> array. The object
+    * is first converted to a <code>Json</code> instance by calling the static
+    * {@link #make} method.
+    * </p>
+    *
+    * @param anything Any Java object that can be converted to a Json instance.
+    * @return this
+    */
+   public final Json add(Object anything) {
+      return add(make(anything));
+   }
+
+   /**
+    * <p>
+    * Remove the specified property from a <code>Json</code> object and return
+    * that property.
+    * </p>
+    *
+    * @param property The property to be removed.
+    * @return The property value or <code>null</code> if the object didn't have such
+    * a property to begin with.
+    */
+   public Json atDel(String property) {
+      throw new UnsupportedOperationException();
+   }
+
+   /**
+    * <p>
+    * Remove the element at the specified index from a <code>Json</code> array and return
+    * that element.
+    * </p>
+    *
+    * @param index The index of the element to delete.
+    * @return The element value.
+    */
+   public Json atDel(int index) {
+      throw new UnsupportedOperationException();
+   }
+
+   /**
+    * <p>
+    * Delete the specified property from a <code>Json</code> object.
+    * </p>
+    *
+    * @param property The property to be removed.
+    * @return this
+    */
+   public Json delAt(String property) {
+      throw new UnsupportedOperationException();
+   }
+
+   /**
+    * <p>
+    * Remove the element at the specified index from a <code>Json</code> array.
+    * </p>
+    *
+    * @param index The index of the element to delete.
+    * @return this
+    */
+   public Json delAt(int index) {
+      throw new UnsupportedOperationException();
+   }
+
+   /**
+    * <p>
+    * Remove the specified element from a <code>Json</code> array.
+    * </p>
+    *
+    * @param el The element to delete.
+    * @return this
+    */
+   public Json remove(Json el) {
+      throw new UnsupportedOperationException();
+   }
+
+   /**
+    * <p>
+    * Remove the specified Java object (converted to a Json instance)
+    * from a <code>Json</code> array. This is equivalent to
+    * <code>remove({@link #make(Object)})</code>.
+    * </p>
+    *
+    * @param anything The object to delete.
+    * @return this
+    */
+   public final Json remove(Object anything) {
+      return remove(make(anything));
+   }
+
+   /**
+    * <p>
+    * Set a <code>Json</code> objects's property.
+    * </p>
+    *
+    * @param property The property name.
+    * @param value The value of the property.
+    * @return this
+    */
+   public Json set(String property, Json value) {
+      throw new UnsupportedOperationException();
+   }
+
+   /**
+    
```

Comment:
Do we need this `main()`?

Reply:
This class is from mJSON library, I avoided changing it to facilitate upgrades in the future.